### PR TITLE
Admin Toolbar > 2.1 requires Drupal 8.8

### PR DESCRIPTION
**Pull request by dbosen**
Make sure these boxes are checked before submitting your pull request - thank you!

- [ ] All coding styles are fulfilled. ([How to check for cs issues?](https://github.com/BurdaMagazinOrg/thunder-dev-tools/blob/master/README.md#code-style-guidelines))
- [ ] All tests are running locally. ([How to run the test?](https://github.com/thunder/thunder-distribution/blob/8.x-4.x/docs/development.md#how-to-run-the-tests))
- [ ] Necessary update hooks are provided.
- [ ] User roles have correct access for new introduced permission.
- [ ] Every thunder module has a README.md in its root. Follow [this guidelines](https://www.drupal.org/node/2181737), but we don't need every topic.
- [ ] Code is covered with well-balanced amount of inline comments.

If you are really awesome, then your feature is covered by additional tests. Well done!

### CODE REVIEW

**Comment by chrfritsch** [comment]
It is since 2.1. So I think we should limit it in the composer.json as well, because the info.yml of admin toolbar 2.1 still states that it is D8.7 compatible.

**Comment by chrfritsch** [comment]
https://www.drupal.org/node/3121816

**Comment by dbosen** [comment]
> It is since 2.1. So I think we should limit it in the composer.json as well, because the info.yml of admin toolbar 2.1 still states that it is D8.7 compatible.

This will lead to awkward situations, where installs will fallback to old Thunder versions, without the restriction...

**Comment by chrfritsch** [comment]
> This will lead to awkward situations, where installs will fallback to old Thunder versions, without the restriction...

I don't think so. Because projects are requiring Thunder, not admin_toolbar. So composer will always try to get the latest version of Thunder

File: composer.json [hunk 1]
```diff
@@ -86,7 +86,7 @@
         "oomphinc/composer-installers-extender": "^1.1",
         "drupal/access_unpublished": "^1.0-alpha8",
         "drupal/adsense": "^1.0",
-        "drupal/admin_toolbar": "^2.0",
+        "drupal/admin_toolbar": "2.0",
```
Comment (chrfritsch) [inline composer.json]: Should this maybe <2.1. If someone is using the beta and only wants to update thunder

Comment (chrfritsch) [reply composer.json]: But I don't have strong opinions on this. You know what I think about people that are using outdated software 😶

**Review by chrfritsch — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (dbosen) [reply composer.json]: I was actually quite shocked, that you brought this up ;-)

**Pull request merged**